src/main/java/com/iexec/common/sdk/order/payload/Order.java

```diff
@@ -37,5 +37,15 @@ public abstract class Order {
     public boolean isSigned() {
         return sign != null && !sign.isEmpty();
     }
+
+    /**
+     * Converts all characters to Lowercase or returns empty
+     * @param s String to convert
+     * @return the String, converted to lowercase or an empty string
+     */
+    public String toLowerCase(String s) {
```

Comment:
Where is it used?

Reply:
In the classes from the same package (`com.iexec.common.sdk.order.payload`) inheriting from Order abstract class

Reply:
I did not find OrderUtils usage in other repositories (blockchain-adapter, result-proxy, sms, core, worker, integration tests, pre/post compute)